main/store/migrations/index.js

```diff
@@ -927,6 +927,17 @@ const migrations = {
       initial.main.shortcuts.summon.enabled = true
     }
 
+    return initial
+  },
+  37: (initial) => {
+    const isWindows = process.platform === 'win32'
+    const { shortcuts } = initial.main || {}
+    const altGrIndex = shortcuts.summon.modifierKeys.indexOf('AltGr')
```

Comment:
is there any chance `modifierKeys` could be undefined here or is it always an array, either empty or otherwise?

Reply:
I checked this on Windows, if you just specify a shortcutKey then modifierKeys is an empty array